environment.md

```diff
@@ -59,23 +75,16 @@ The priority of the transports are Direct file IO, SSH2, FTP PHP Extension, FTP
 
 ## Database
 
-WordPress stores content, configuration, and other information inside of a database. The database for a WordPress website is where all of the WordPress website's user-defined data is stored. WordPress is primarily designed to use a MySQL or MySQL-related database server; WordPress officially only supports MySQL or MariaDB, a drop-in replacement for MySQL.
-
-### MySQL
-
-MySQL is a widely used relational database server. MySQL comes in both open-source and commercial distributions. Either distribution should work with MySQL. The commercial distribution has additional features not found in the open-source distribution; however, WordPress does not require or use the additional features. It is designed to run on either the open-source or the commercial distribution.
-
-### MariaDB
-
-MariaDB is a drop-in replacement for MySQL supported by WordPress. It's a fork of the open-source distribution of MySQL. MariaDB was originally created to maintain a more open-source version of MySQL, but it has grown into its own relational database server alternative to MySQL with features and changes not found in MySQL. Despite its differences, MariaDB is still a fully compatible replacement for MySQL and can generally seamlessly replace MySQL.
+For data storage, WordPress uses systems based on MySQL distributions, such as MySQL, MariaDB or Percona. All three systems are compatible with WordPress, so at this moment any of the three can be used for its operation.
```

Comment:
WordPress doesn't officially support Percona. I think it'd make sense to find out why before adding it. Might be a good separate issue, unless it's easy to track down?
`https://wordpress.org/about/requirements/`

Reply:
I've been using Percona a lot of years ago with WordPress without any problem... I cannot find information about incompatibility issues... :(

Reply:
I think we should check with folks about this, since we shouldn't recommend software that isn't supported by WordPress.